modules/infra/control/nexthop.c

```diff
@@ -192,7 +191,6 @@ void nexthop_decref(struct nexthop *nh) {
 		}
 		rte_mempool_put(nhp->mp, nh);
 		memset(nh, 0, sizeof(*nh));
-		rte_spinlock_unlock(&nh->lock);
```

Comment:
Good thing we are removing this spinlock as it was buggy here:
- previous line memset released the lock,
- a concurrent thread spinning would acquire the lock,
- this code then would incorrectly release the lock,